automation/taskcluster/lib/tasks.py

```diff
@@ -144,6 +158,9 @@ def craft_assemble_task(self, variant):
                 'symbol': 'A',
                 'tier': 1,
             },
+            scopes=[
+                "secrets:get:project/mobile/fenix/{}".format(variant.build_type)
```

Comment:
I don't think that separating by build type quite makes sense anymore.
Before, we used to just have secrets for each of the release channels (`production`/`beta`/`nightly`), in which case each of the release build types matched up one-to-one.
But, we are starting to see some PR-specific secrets, like this codecov secret and the nimbledroid secret.

I'm guessing that our secrets now fall into one of the following four buckets:
* `nightly`
* `beta`
* `production`
* `pr`

Reply:
Updated.